### DISCUSSION TITLE
How json data is encoded ?

### DISCUSSION THREAD

**Discussion by Barnoux** [dani-garcia/vaultwarden#3772] [Q&A] [answered]
Hello,

I attempted to examine the source code of Vaultwarden in search of an answer, but unfortunately, I haven't been able to locate the solution I was seeking.

Specifically, I'm trying to understand the encoding method employed for the JSON data that the client sends when generating a new element. A pertinent example pertains to the 'name' field contained within the JSON structure.

By utilizing the developer tools, I've managed to identify a file labeled as 'encrypt-worker.*.js'. It appears highly probable that this file holds the key to the information I'm in pursuit of.

Regards,

BBA

**Comment by BlackDex** [comment, marked as answer]
You are talking about the web-vault code, not the server code.

The web-vault code is maintained by Bitwarden, we only use it and apply some patches to make it work with the Vaultwarden backend.
The web-vault code can be found here: https://github.com/bitwarden/clients
